src/main/java/seedu/eduke8/Eduke8.java

```diff
@@ -24,14 +26,15 @@
     private static final String LOG_PATH = "data/logs/" + DATE_TIME_NOW.format(DATE_TIME_FORMATTER) + ".log";
     private static final Logger LOGGER = Logger.getLogger(Logger.GLOBAL_LOGGER_NAME);
 
+    public BookmarkList bookmarks = new BookmarkList();
```

Comment:
I think this is ok for now, i will add in a storage system to load the bookmarks along with notes